site/assets/scss/app.scss

```diff
@@ -982,3 +982,65 @@ a.classifiation-tag:hover {
     }
   }
 }
+
+// Notification Template
+.notify-drop-footer{
+  border-top: 2px solid #e2e2e2;
+}
+.notify-drop{
+  min-width: 330px;
+  background-color: #fff;
+  min-height: 280px;
+  max-height: 280px;
+}
+.notify-drop .notify-drop-title {
+  border-bottom: 2px solid #e2e2e2;
+  padding: 5px 15px 10px 15px;
+}
+.notify-drop .notify-drop-title .notification-title{
+  font-weight: bold;
+}
+.notify-drop .drop-content {
+  min-height: 200px;
+  max-height: 200px;
+  overflow-y: scroll;
+}
+.notify-drop .drop-content::-webkit-scrollbar-track {
+  background-color: #F5F5F5;
+}
+.notify-drop .drop-content::-webkit-scrollbar {
+  width: 8px;
+  background-color: #F5F5F5;
+}
+.notify-drop .drop-content::-webkit-scrollbar-thumb {
+  background-color: #ccc;
+}
+.notification-unseen {
```

Comment:
For all these .notification-* classes: Use a single class like `notification` on the outer element (describing a single notification), and then only `unseen`, `seen`, etc. on the sub-elements. You can then do `.notification .unseen`, or nested blocks (it's SASS).